db/fixtures/miq_product_features.yml

```diff
@@ -3415,6 +3415,10 @@
       :description: Display Individual Middleware Datasources
       :feature_type: view
       :identifier: middleware_datasource_show
+    - :name: Utilization
```

Comment:
I think you should update the features count in the test otherwise it will fail (or already fails :) )